binding.gyp

```diff
@@ -24,12 +21,33 @@
             "conditions": [
                 [ "SASL==\"n\"", { "sources!": 
                   ["LDAPSASL.cc", "SASLDefaults.cc"] } ], 
-                [ "SASL==\"y\"", { "sources!": ["LDAPXSASL.cc"] } ]
+                [ "SASL==\"y\"", { "sources!": ["LDAPXSASL.cc"] } ],
+		['NODE_VERSION > 9', {
+                  "conditions": [[
+                    'OS=="linux"', {
+                      "conditions": [[
+                        "REDHAT_RELEASE == 6", {
+                          "libraries": [ "../deps/RHEL6/libldap.a", "../deps/RHEL6/liblber.a" ]
+                        }, {
+                          "libraries": [ "../deps/RHEL7/libldap.a", "../deps/RHEL7/liblber.a" ]
+                        }
+                      ]]
+                    }, {
+                      "libraries": [ "../deps/OSX/libldap.a", "../deps/OSX/liblber.a" ],
+                    }
+                  ]],
+                  "include_dirs": [ "deps/include" ],
+                  "libraries": [ "-lresolv", "-lsasl2" ]
```

Comment:
Might be a bug here, it looks like this forces you to have the sasl library installed I'll come back to this later